api/src/main/java/io/opentelemetry/trace/TracingContextUtils.java

```diff
@@ -30,7 +30,21 @@
 @Immutable
 public final class TracingContextUtils {
   private static final Context.Key<Span> CONTEXT_SPAN_KEY =
-      Context.<Span>key("opentelemetry-trace-span-key");
+      Context.<Span>keyWithDefault("opentelemetry-trace-span-key", DefaultSpan.getInvalid());
+  private static final Context.Key<SpanContext> CONTEXT_SPANCONTEXT_KEY =
+      Context.<SpanContext>keyWithDefault(
+          "opentelemetry-trace-spancontext-key", SpanContext.getInvalid());
```

Comment:
Let's use the DefaultSpan to encapsulate the received Span. In the Builder we can check if the "isRemote()" is true to distinguish.

Reply:
Let's take a roll at that. I'm still wondering about the potential effects of it, but I guess we will know when beta hits the users. In that case I will remove the utils part related to `SpanContext` (that's more work from my side but hopefully it will help get faster reviews).